src/helpers/EngineerHelper.js

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from "react";
+import utils from "./Utils";
+import restApiHelper from "./RestApiHelper";
+const useEngineers = () => {
+  const [engineers, setEngineers] = useState([]);
+  useEffect(() => {
+    getEngineersLocally();
+  }, []);
+
+  const getEngineersLocally = () => {
+    const _engineers = restApiHelper._getLocalItems("engineers");
```

Comment:
the `_` in `_getLocalItems` means that the function is private and not used ouside the file it was written in. So this is wrong.

Also restApiHelper should return functions such as getEngineers, getPapers etc. not getLocalItems.